variables.tf

```diff
@@ -1,51 +1,43 @@
-variable "triton_account" {
-    default = "<username>"
-}
-
-variable "triton_key_id" {
-    default = "<fingerprint>"
+#
+# Details about all deployments of this application
+#
+variable "production" {
+  type = "string"
+  default = "blue"
 }
 
-variable "dc_name" {
-    default = "<dc_name>"
+variable "service_name" {
+	default = "happiness"
 }
 
-variable "production" {
-  type = "string"
-  default = "blue"
+#
+# Details about the "blue" deployment
+#
+variable "blue_image_name" {
+    default = "ubuntu-16.04"
 }
 
-variable "staging" {
```

Comment:
I'm eager to remove this var for a bunch of reasons, but mostly so that we have one and only one switch to flip the blue and green deployments between staging and production.